volume/volume_windows.go

```diff
@@ -199,3 +205,25 @@ func validateStat(fi os.FileInfo) error {
 	}
 	return nil
 }
+
+func validateAbsolute(p string) error {
```

Comment:
Yes, after having seen CI test failures, I agree that we must do different parsing / validation logic to handle linux containers and windows containers.
However, we can´t leverage existing linux parsing/validation completely, as the source path is still windows style, and the linux parsing code is based on the assumption that `:` is a forbidden character in a file path...